src/SFA.DAS.Apprenticeships.Api.Types/ApprenticeshipSearchResultsItem.cs

```diff
@@ -34,5 +34,7 @@ public class ApprenticeshipSearchResultsItem
 		public DateTime? EffectiveFrom { get; set; }
 
 		public DateTime? EffectiveTo { get; set; }
-	}
+
+        public DateTime? LastDateForNewStarts { get; set; }
+    }
 }
```

Comment:
as we are making breaking changes and we know frameworks are being deprecated, is it worth taking the opportunity to standardise the return items? there is a lot of framework specific naming conventions which dont really make sense in the 'new work'? Also it may be better to have a single Id field and an apprenticeship type to differentiate between framework and standards

Reply:
That assumes that no consumers would want framework specific information such as pathway name, maybe!.
Frameworks are still going to be around for the next year at least so it's not like they're going anywhere any time soon.

Reply:
totally agree now I've had another look, however having two Id fields still feels a bit wrong to me and my main issue with how it is currently.

Reply:
This has been updated to have a different result item structure.